test/AutoDiff/simple_math.swift

```diff
@@ -350,4 +351,46 @@ SimpleMathTests.test("ForceUnwrapping") {
   expectEqual((1, 2), forceUnwrap(Float(2)))
 }
 
+// CHECK-LABEL: sil hidden [ossa] @AD__${{.*}}jumpTimesTwo{{.*}}pullback_src_0_wrt_0 : $@convention(thin) (Float, @owned _AD__$s4nullyycfU18_12jumpTimesTwoL_5modelSfAAyycfU18_14SmallTestModelL_V_tF_bb0__PB__src_0_wrt_0) -> SmallTestModel.TangentVector {
```

Comment:
FileCheck'ing only a few SIL lines is sufficient! The most important line is the accumulation `apply +=`; most of the rest (`destroy_value`, `destroy_addr`, `dealloc_stack`) can be removed.